rust/xaynet-server/src/settings/mod.rs

```diff
@@ -452,7 +455,7 @@ impl From<MaskSettings> for MaskConfig {
     }
 }
 
-#[derive(Debug, Deserialize)]
+#[derive(Debug, Deserialize, Copy, Clone)]
```

Comment:
I'd rather not derive `Copy` here, because it has little benefit and me may forget to remove when we add more settings.